app/src/main/java/com/hedvig/app/feature/claims/ui/ClaimsFragment.kt

```diff
@@ -38,10 +39,14 @@ class ClaimsFragment : BaseTabFragment() {
     private val requestBuilder: RequestBuilder<PictureDrawable> by lazy { buildRequestBuilder() }
     private val baseMargin: Int by lazy { resources.getDimensionPixelSize(R.dimen.base_margin) }
 
+    private var toolbar: androidx.appcompat.widget.Toolbar? = null
     override val layout = R.layout.fragment_claims
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        toolbar = activity?.findViewById(R.id.hedvigToolbar)
```

Comment:
Is it possible to get a view from the parent activity in a fragment? When i try i get `null`

Reply:
Generally, I think there are some architectural problems with accessing views in a parent `Activity` from a child `Fragment`: The parent `Activity` should probably have ownership of its own `View`s. Perhaps this could be resolved by using a `ViewModel` that the parent and child shares? That could facilitate the communication between the two units.